command/kv_put.go

```diff
@@ -229,8 +229,12 @@ func (c *KVPutCommand) dataFromArgs(args []string) (string, string, error) {
 		return key, string(data), nil
 	case '-':
 		var b bytes.Buffer
-		if _, err := io.Copy(&b, stdin); err != nil {
-			return "", "", fmt.Errorf("Failed to read stdin: %s", err)
+		if len(data) > 1 {
+			return key, data, nil
+		} else {
+			if _, err := io.Copy(&b, stdin); err != nil {
```

Comment:
We should move `var b bytes.Buffer` down above the `io.Copy()` call since it's only ever referenced in the `else` clause.